pkg/smokescreen/config.go

```diff
@@ -13,18 +13,24 @@ import (
 	"net"
 	"net/http"
 	"os"
+	"strconv"
 	"sync"
 	"time"
 
 	"github.com/DataDog/datadog-go/statsd"
 	log "github.com/sirupsen/logrus"
 )
 
+type RuleRange struct {
```

Comment:
I think `RuleRange` is kind of a weak name. Open to ideas.

Reply:
`RangeWithPort` maybe?  I'm pretty ok with `RuleRange` though.

Reply:
I can leave as-is if you think it's OK. Sometimes I over-think the naming...